[2024-02-11T03:11:05.949+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T03:10:39.349014+00:00 [queued]>
[2024-02-11T03:11:05.959+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T03:10:39.349014+00:00 [queued]>
[2024-02-11T03:11:05.960+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-11T03:11:05.978+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): rename_censo_dir> on 2024-02-11 03:10:39.349014+00:00
[2024-02-11T03:11:05.986+0000] {standard_task_runner.py:60} INFO - Started process 338 to run task
[2024-02-11T03:11:05.989+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'censoEnade', 'rename_censo_dir', 'manual__2024-02-11T03:10:39.349014+00:00', '--job-id', '609', '--raw', '--subdir', 'DAGS_FOLDER/censoEnade.py', '--cfg-path', '/tmp/tmp9w_2y7m9']
[2024-02-11T03:11:05.990+0000] {standard_task_runner.py:88} INFO - Job 609: Subtask rename_censo_dir
[2024-02-11T03:11:06.043+0000] {task_command.py:423} INFO - Running <TaskInstance: censoEnade.rename_censo_dir manual__2024-02-11T03:10:39.349014+00:00 [running]> on host 19f7d6a2ee45
[2024-02-11T03:11:06.143+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Néfi Fernandes' AIRFLOW_CTX_DAG_ID='censoEnade' AIRFLOW_CTX_TASK_ID='rename_censo_dir' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T03:10:39.349014+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-11T03:10:39.349014+00:00'
[2024-02-11T03:11:06.145+0000] {logging_mixin.py:188} INFO - O diretório censo_ed_superior_2021 já existe. Continuando para a próxima etapa.
[2024-02-11T03:11:06.146+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-11T03:11:06.157+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=censoEnade, task_id=rename_censo_dir, execution_date=20240211T031039, start_date=20240211T031105, end_date=20240211T031106
[2024-02-11T03:11:06.202+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-11T03:11:06.224+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
